lib/components.js

```diff
@@ -363,7 +363,9 @@ class DatePicker extends Component {
       'minimumDate',
       'minuteInterval',
       'mode',
-      'timeZoneOffsetInMinutes'
+      'timeZoneOffsetInMinutes',
+      'background',
```

Comment:
No, your point does make sense and I prefer to discuss things before implementing it in a wrong way.

I'll move the prop into config, and take out of there the `mode` tomorrow.